### DISCUSSION TITLE
how can i uninstal the program?

### DISCUSSION THREAD

**Discussion by midas6142** [Reloaded-Project/Reloaded-II#476] [Q&A]
seems like a stupid question but I really don't know how to uninstall the program.

**Comment by Sewer56** [comment]
Removing the Reloaded-II folder from your desktop is normally sufficient to do the job.

If you've installed ASI Loader before, then you should also remove Reloaded.Mod.Bootstrapper.asi from the game folder.

------------

If you want to do full cleanup, forgetting all preferences, you can also remove the `Reloaded-Mod-Loader-II` folder from `AppData/Roaming` and `AppData/Local`.
